src/Install/DatabaseConfig.php

```diff
@@ -47,7 +47,7 @@ public function toArray()
             'collation' => 'utf8mb4_unicode_ci',
             'prefix'    => $this->prefix,
             'strict'    => false,
-            'engine'    => 'InnoDB',
+            'engine'    => null,
```

Comment:
Should we even specify this key if we're just setting it to null?

Reply:
That's the convention in Laravel,  yes.